### ISSUE TITLE
Can't run btdu on ubuntu 22.04 wsl 2

### ISSUE THREAD

**Issue by santagada** [CyberShadow/btdu#24]
```
btdu .
Fatal error: Unexpected ';' when converting from type const(char)[] to type int
```

I think btdu is looking at some windows environment variable or something, those are separated with ;. Without what  the env is I can't help more I think

**Comment by santagada** [comment]
this is running latest static linked release binary

**Comment by santagada** [comment]
```
# cat /proc/self/mounts
none /mnt/wsl tmpfs rw,relatime 0 0
none /usr/lib/wsl/drivers 9p ro,dirsync,nosuid,nodev,noatime,aname=drivers;fmask=222;dmask=222,mmap,access=client,msize=65536,trans=fd,rfd=7,wfd=7 0 0
none /usr/lib/wsl/lib overlay rw,relatime,lowerdir=/gpu_lib_packaged:/gpu_lib_inbox,upperdir=/gpu_lib/rw/upper,workdir=/gpu_lib/rw/work 0 0
/dev/sdc / ext4 rw,relatime,discard,errors=remount-ro,data=ordered 0 0
none /mnt/wslg tmpfs rw,relatime 0 0
/dev/sdc /mnt/wslg/distro ext4 ro,relatime,discard,errors=remount-ro,data=ordered 0 0
rootfs /init rootfs ro,size=32839632k,nr_inodes=8209908 0 0
none /dev devtmpfs rw,nosuid,relatime,size=32839664k,nr_inodes=8209916,mode=755 0 0
sysfs /sys sysfs rw,nosuid,nodev,noexec,noatime 0 0
proc /proc proc rw,nosuid,nodev,noexec,noatime 0 0
devpts /dev/pts devpts rw,nosuid,noexec,noatime,gid=5,mode=620,ptmxmode=000 0 0
none /run tmpfs rw,nosuid,nodev,mode=755 0 0
none /run/lock tmpfs rw,nosuid,nodev,noexec,noatime 0 0
none /run/shm tmpfs rw,nosuid,nodev,noatime 0 0
none /dev/shm tmpfs rw,nosuid,nodev,noatime 0 0
none /run/user tmpfs rw,nosuid,nodev,noexec,noatime,mode=755 0 0
binfmt_misc /proc/sys/fs/binfmt_misc binfmt_misc rw,relatime 0 0
tmpfs /sys/fs/cgroup tmpfs ro,nosuid,nodev,noexec,size=4096k,nr_inodes=1024,mode=755 0 0
cgroup2 /sys/fs/cgroup/unified cgroup2 rw,nosuid,nodev,noexec,relatime,nsdelegate 0 0
cgroup /sys/fs/cgroup/cpuset cgroup rw,nosuid,nodev,noexec,relatime,cpuset 0 0
cgroup /sys/fs/cgroup/cpu cgroup rw,nosuid,nodev,noexec,relatime,cpu 0 0
cgroup /sys/fs/cgroup/cpuacct cgroup rw,nosuid,nodev,noexec,relatime,cpuacct 0 0
cgroup /sys/fs/cgroup/blkio cgroup rw,nosuid,nodev,noexec,relatime,blkio 0 0
cgroup /sys/fs/cgroup/memory cgroup rw,nosuid,nodev,noexec,relatime,memory 0 0
cgroup /sys/fs/cgroup/devices cgroup rw,nosuid,nodev,noexec,relatime,devices 0 0
cgroup /sys/fs/cgroup/freezer cgroup rw,nosuid,nodev,noexec,relatime,freezer 0 0
cgroup /sys/fs/cgroup/net_cls cgroup rw,nosuid,nodev,noexec,relatime,net_cls 0 0
cgroup /sys/fs/cgroup/perf_event cgroup rw,nosuid,nodev,noexec,relatime,perf_event 0 0
cgroup /sys/fs/cgroup/net_prio cgroup rw,nosuid,nodev,noexec,relatime,net_prio 0 0
cgroup /sys/fs/cgroup/hugetlb cgroup rw,nosuid,nodev,noexec,relatime,hugetlb 0 0
cgroup /sys/fs/cgroup/pids cgroup rw,nosuid,nodev,noexec,relatime,pids 0 0
cgroup /sys/fs/cgroup/rdma cgroup rw,nosuid,nodev,noexec,relatime,rdma 0 0
cgroup /sys/fs/cgroup/misc cgroup rw,nosuid,nodev,noexec,relatime,misc 0 0
none /mnt/wslg/versions.txt overlay rw,relatime,lowerdir=/systemvhd,upperdir=/system/rw/upper,workdir=/system/rw/work 0 0
none /mnt/wslg/doc overlay rw,relatime,lowerdir=/systemvhd,upperdir=/system/rw/upper,workdir=/system/rw/work 0 0
none /tmp/.X11-unix tmpfs ro,relatime 0 0
drvfs /mnt/c 9p rw,dirsync,noatime,aname=drvfs;path=C:\;uid=1000;gid=1000;symlinkroot=/mnt/,mmap,access=client,msize=262144,trans=virtio 0 0
drvfs /mnt/d 9p rw,dirsync,noatime,aname=drvfs;path=D:\;uid=1000;gid=1000;symlinkroot=/mnt/,mmap,access=client,msize=262144,trans=virtio 0 0
drvfs /mnt/e 9p rw,dirsync,noatime,aname=drvfs;path=E:\;uid=1000;gid=1000;symlinkroot=/mnt/,mmap,access=client,msize=262144,trans=virtio 0 0
drvfs /mnt/g 9p rw,dirsync,noatime,aname=drvfs;path=G:\;uid=1000;gid=1000;symlinkroot=/mnt/,mmap,access=client,msize=262144,trans=virtio 0 0
cgroup /sys/fs/cgroup/systemd cgroup rw,nosuid,nodev,noexec,relatime,xattr,name=systemd 0 0
hugetlbfs /dev/hugepages hugetlbfs rw,relatime,pagesize=2M 0 0
mqueue /dev/mqueue mqueue rw,nosuid,nodev,noexec,relatime 0 0
debugfs /sys/kernel/debug debugfs rw,nosuid,nodev,noexec,relatime 0 0
tracefs /sys/kernel/tracing tracefs rw,nosuid,nodev,noexec,relatime 0 0
fusectl /sys/fs/fuse/connections fusectl rw,nosuid,nodev,noexec,relatime 0 0
/dev/sdc /snap ext4 rw,relatime,discard,errors=remount-ro,data=ordered 0 0
snapfuse /snap/bare/5 fuse.snapfuse ro,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
snapfuse /snap/core22/607 fuse.snapfuse ro,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
snapfuse /snap/core22/858 fuse.snapfuse ro,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
snapfuse /snap/gtk-common-themes/1535 fuse.snapfuse ro,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
snapfuse /snap/snapd/19457 fuse.snapfuse ro,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
none /run/user/1000 tmpfs rw,relatime 0 0
/dev/sdd2 /mnt/test ext4 rw,relatime 0 0
tracefs /sys/kernel/debug/tracing tracefs rw,nosuid,nodev,noexec,relatime 0 0
snapfuse /snap/ubuntu-desktop-installer/1205 fuse.snapfuse ro,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
tmpfs /run/user/0 tmpfs rw,nosuid,nodev,relatime,size=6568576k,nr_inodes=1642144,mode=700 0 0
/dev/sdd3 /mnt/btrfs btrfs rw,nodev,noatime,nodatasum,nodatacow,ssd,space_cache=v2,subvolid=5,subvol=/ 0 0
snapfuse /snap/ubuntu-desktop-installer/1207 fuse.snapfuse ro,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
snapfuse /snap/snapd/19993 fuse.snapfuse ro,nodev,relatime,user_id=0,group_id=0,allow_other 0 0
```

**Comment by CyberShadow** [comment]
Thanks, fixed in 474bc736ac59ec503a7d8f96b7265fc46f348b88 and 76b07f94628a3b3e38578264412e55a6126c57a4.
